README.md

```diff
@@ -3,25 +3,34 @@
 [![Github-CI][github-ci]][github-link]
 [![Documentation Status][rtd-badge]][rtd-link]
 
-**THIS IS A DRAFT (i.e., WORK IN PROGRESS)**
-
 
 This repository contains the source files for the Research Computing documentation.
 
 HPC users can submit issues and bugs in the documentation {here}.
 
 
 ## Development
-Clone repo and create python environment:
+Clone repo:
 
 ```bash
 git clone https://github.com/northeastern-rc/rc-public-documentation
+```
+Then, either open PyCharm and create and create python environment:
```

Comment:
Typo fix, change to -> "Then, either open PyCharm and create python environment"